acdh-schema.owl

```diff
@@ -161,6 +161,36 @@ Domain für acdh:hasFilename geändert (um acdh:Collection und acdh:TopCollectio
 
 Kardinalität von acdh:isTitleImageOf für acdh:Resource geändert zu 0-n, um die mehrfache Nutzung von Bildern zu erlauben.
 Kardinalität von acdh:hasCategory für acdh:BinaryContent gändert zu 0-n, um mehrere Kategorien zu erlauben.</owl:versionInfo>
+        <owl:versionInfo xml:lang="en">Version 4.0 released on 2023-12-22.
+Renamed to ARCHE Schema (was ACDH Schema before)
+Introduced annotation property acdh:exampleValue to allow for the inclusion of example values.
+Introduced acdh:hasNextItem with domain acdh:RepoObject.
+Introduced acdh:hasIiifManifest with domain acdh:BinaryContent.
+Introduced acdh:hasRightsInfomation with domain acdh:RepoObject.
+Expanded description for acdh:hasOaiSet to include visibility in OpenAIRE.
+
+Dropped acdh:hasLocationPath as it wasn&apos;t technically required anymore.
+
+Added acdh:automatedFill &apos;true&apos; to acdh:disseminationServices.
```

Comment:
`acdh:automatedFill` for `acdh:disseminationServices` as well as for `acdh:technical` was already introduced in [version 3.1.2](https://github.com/acdh-oeaw/arche-schema/releases/tag/v3.1.2), so we could add a new versionInfo related to 3.1.2 and mention this there.

Reply:
Since it was a minor change anyway I think it is okay to leave it as it is